packages/web/server/EventKit.ts

```diff
@@ -0,0 +1,70 @@
+import { Attachment, FieldSet, Table } from 'airtable'
+import airtableInit from './airtable'
+import cache from './cache'
+
+const BASE_ID = 'appjKfoHvrO5SZWdd'
```

Comment:
What's this?

Reply:
the id for the  airtable base. I can rename for clarity